docs/source/tutorial/function.rst

```diff
@@ -608,4 +608,4 @@ offers a utility function :func:`chainer.gradient_check.check_backward` that fol
    unittest.TextTestRunner().run(suite)
 
 
-You can find many examples of function tests under ``tests/chainer_tests/function_tests`` directory.
+You can find many examples of function tests under :tree:`tests/chainer_tests/function_tests` directory.
```

Comment:
`function_tests` -> `functions_tests`

Reply:
I appreciate it. I should scrutinize my code more carefully.